Cargo.lock

```diff
@@ -2356,16 +2357,6 @@ version = "1.15.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "67b1b7a3b5fe4f1376887184045fcf45c69e92af734b7aaddc05fb777b6fbd03"
 
-[[package]]
-name = "socket2"
-version = "0.5.10"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "e22376abed350d73dd1cd119b57ffccad95b4e585a7cda43e286245ce23c0678"
-dependencies = [
- "libc",
- "windows-sys 0.52.0",
-]
-
```

Comment:
We should update deny.toml to remove the exception https://github.com/linkerd/linkerd2/blob/224693ed02dad1bdeac07102d0ad332fa3e81490/deny.toml#L68-L69